examples/images_contours_and_fields/image_antialiasing.py

```diff
@@ -72,6 +74,73 @@
     ax.set_title(f"interpolation='{interp}'")
 plt.show()
 
+###############################################################################
+# Data antialiasing and RGBA antialiasing
+# ----------------------------------------
+#
+# The examples above all used grayscale.  When colormapping is added there
+# is the complication that downsampling and the antialiasing filters are
+# applied to the data in Matplotlib, before the data is mapped to
+# colors.  So in the following note how the corners fade to white, the middle
+# of the colormap, because the data is being smoothed and the high and low
+# values are averaging out to the middle of the colormap.
+
+f0 = 10
+k = 150
+a = np.sin(np.pi * 2 * (f0 * R + k * R**2 / 2))
+
+fig, axs = plt.subplots(1, 2, figsize=(7, 4), sharex=True, sharey=True,
+                        constrained_layout=True)
+for ax, interp in zip(axs, ['nearest', 'antialiased']):
+    pc = ax.imshow(a, interpolation=interp, cmap='RdBu_r', vmin=-1, vmax=1)
+    ax.set_title(f"'{interp}'", fontsize='small')
```

Comment:
```suggestion
    ax.set_title(f"'{interp}'")
```
I would not reduce the title size:
- The titles are important for distinction if you only look at the image.
- Consistency: All titles above use the original title size.